src/main/java/lotto/view/ResultView.java

```diff
@@ -51,12 +49,16 @@ public static void showGeneratedLottos(Lottos lottos) {
 
     private static String createLottoForm(Lotto lotto) {
         String result = String.join(
-                ", ",
-                lotto.getLotto()
+            ", ",
+            lotto.getLotto()
                 .stream()
                 .map(lottoNumber -> String.valueOf(lottoNumber.getValue()))
```

Comment:
```suggestion
                .map(LottoNumber::stringValue)
```

LottoNumber가 문자열을 변환해주면 값을 꺼내와서 변환시키는(`get-`)을 안써도 되겠네요.

ex)
<img width="510" alt="스크린샷 2024-03-27 오전 12 00 50" src="https://github.com/next-step/java-lotto/assets/12059592/b0192100-6d05-4d4e-8a41-f924d6c48124">